source/Outfit.cpp

```diff
@@ -286,6 +288,9 @@ void Outfit::Load(const DataNode &node)
 			child.PrintTrace("Skipping unrecognized attribute:");
 	}
 
+	if(pluralName.empty())
+		pluralName = displayName + 's';
```

Comment:
Warp is right, this won't work if you want to have an outfit with no plural name. So how about this: Set the plural name at the start (line 192) (but add a check for `isDefined` so that we don't always override the display/plural name). Then have a bool called `hasPluralName` that gets set if the plural name is explicitly specified and then when specifying a display name check if `hasPluralName` is set and if it is not set the default plural name.